### ISSUE TITLE
Broken link

### ISSUE THREAD

**Issue by rahji** [FSUdigitalmedia/tiny-bitsy-game-aeredd#1]
Hey Avery, your link doesn't work. I think you still need to setup Github Pages and/or update the URL in your readme. (Sorry if I'm jumping the gun and you're already working on it - I just see that there is an HTML file there but no working link to it)

**Comment by aeredd** [comment]
Yes! I’m trying to figure it out.. the way we did the interactive narrative doesn’t work for this one. Trying to trouble shoot it. Kind of confused!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by rahji** [comment]
You should be able to go to the settings for the repo and add github pages (either on the root, if the index.html is there, or a docs folder, if that's where the index.html file is. Then it will show you what the link is.

**Comment by aeredd** [comment]
That's what I did. Still did not work. When I try and upload a docs/mygame this is what pops up.
________________________________
